rfcs/any.js-globals.md

```diff
@@ -0,0 +1,14 @@
+# RFC ..: Remove default and negated global support in *.any.js
+
+## Summary
+
+https://web-platform-tests.org/writing-tests/testharness.html#multi-global-tests has a confusing syntax for specifying globals. E.g., `global=window` is equivalent to `global=window,dedicatedworker` and `global=jsshell` is equivalent to `global=window,dedicatedworker,jsshell`.
+
```

Comment:
Can you add a paragraph on what the new world will look like? (That is, what this RFC will actually change). E.g.

1. globals must be listed explicitly  for the test be run in them
1. if no global meta is specified then the default will apply (maybe specify exactly will the default will be)
1. some change involving negation? I'm not actually sure what you intend to change there.